.github/workflows/ci.yml

```diff
@@ -130,16 +130,18 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
+        poetry-version: ["1.5.1"]
```

Comment:
do you expect to parametrize test by poetry? why not pin it?

Reply:
This is the act of pinning it, without this its `latest`